packages/editor/src/components/hierarchy/HierarchyPanelContainer.tsx

```diff
@@ -483,11 +488,40 @@ function HierarchyPanelContents({ rootEntityUUID }: { rootEntityUUID: EntityUUID
             fontSize: '12px',
             lineHeight: '0.5'
           }}
-          onClick={() => EditorControlFunctions.createObjectFromSceneElement()}
+          onClick={(event) => {
+            anchorElement.set(event.currentTarget)
+            if (getMutableState(EditorHelperState).studioMode.value == 'Advanced') {
+              EditorControlFunctions.createObjectFromSceneElement()
+            }
+          }}
         >
           {t('editor:hierarchy.lbl-addEntity')}
         </PropertiesPanelButton>
       </div>
+      <PopoverContext.Provider
+        value={{
+          handlePopoverClose: () => {
+            anchorElement.set(null)
+          }
+        }}
+      >
+        <Popover
+          id={open ? 'add-component-popover' : undefined}
+          open={open && getMutableState(EditorHelperState).studioMode.value == 'Simple'}
```

Comment:
we need to use a useHookstate or useMutableState hook here instead of just a getMutableState, otherwise this is not reactive